plugins/vpc-branch-eni/plugin/commands.go

```diff
@@ -296,6 +294,24 @@ func (plugin *Plugin) createVLANLink(branch *eni.Branch, linkName string, ipAddr
 	return nil
 }
 
+func getGatewayIPAddress(ipAddress *net.IPNet, gatewayIPAddress net.IP) (net.IP, error) {
+	if gatewayIPAddress != nil {
+		log.Infof("Using gateway IP address specified in NetConfig: %s", gatewayIPAddress.String())
```

Comment:
Echoing https://github.com/aws/amazon-vpc-cni-plugins/pull/3#discussion_r260906937, I don't think we need this log line. Just a log line at the end of the method that prints the `gatewayIPAddress` that's going to be used should suffice.

Reply:
updated